docs/Account.md

```diff
@@ -243,54 +244,57 @@ end
 
 Returns the public key associated with the Account contract.
 
-##### Parameters:
+##### Parameters
```

Comment:
I know the linter doesn't like it, but I believe we should keep the colon after `Parameters` and `Returns`.